lib/alephant/lookup/lookup_cache.rb

```diff
@@ -0,0 +1,67 @@
+require 'dalli-elasticache'
```

Comment:
I copied over this cache class from the Alephant-Broker repo and made some modifications.
> https://github.com/BBC-News/alephant-broker/blob/master/lib/alephant/broker/cache.rb